ScalantCommunity/ScalantLinkTree
# [Enhancement]: Separated links to another data file #2

The links is separated to the data.js file.

enhancement #2 

How should this be tested?
1. Go to code folder
2.  there is new file "data.js" in src folder.
![data](https://github.com/ScalantCommunity/ScalantLinkTree/assets/94776274/649901a7-0aa6-4691-98c5-98886d3ae2c1)

3. See Like this in the code
![show links](https://github.com/ScalantCommunity/ScalantLinkTree/assets/94776274/29be739a-5c74-4f2d-aeb5-f9f3cc5d825a)
